deps/llvm.mk

```diff
@@ -461,7 +463,8 @@ endif
 $(build_prefix)/manifest/llvm: | $(llvm_python_workaround)
 
 LLVM_INSTALL = \
-	cd $1 && $$(CMAKE) -DCMAKE_INSTALL_PREFIX="$2$$(build_prefix)" -P cmake_install.cmake
+	cd $1 && cp -r $$(LLVM_SRC_DIR)/utils/lit $2$$(build_depsbindir) && \
```

Comment:
@vtjnash is there a more elegant way of doing this?

Reply:
Such as?